packages/salesforcedx-vscode-soql/src/queryDataView/queryDataViewService.ts

```diff
@@ -64,7 +64,7 @@ export class QueryDataViewService {
         const errorType = 'data_view_post_message';
         const message = nls.localize('error_unknown_error', errorType);
         channelService.appendLine(message);
-        trackErrorWithTelemetry(errorType, err);
+        await trackErrorWithTelemetry(errorType, err);
```

Comment:
`trackErrorWithTelemetry()` isn't an async function.

```suggestion
        trackErrorWithTelemetry(errorType, err);
```

Reply:
I think the only reason it was was to enable `.catch(console.error)` in queryDataViewService.  

so I wrapped it in another try catch so the console.error happens with it having to be that weird faux-async